snowflake_stage_grant/main.tf.json

```diff
@@ -0,0 +1,67 @@
+{
+  "//": "NOTE: Generated by scripts/snowflake_generate_grant_all.\n\tChanges made directly to this file will be overwritten.\n\tMake improvements there so everyone can benefit.\n  The reason this module exists is that the provider only supports one grant resource per (database_name, schema_name, table_name, on_future, with_grant_option) tuple.\n\tFor example, if you used this module to grant an ALL privilege to a role you couldn't grant a subset of the ALL privs to another role.",
+  "variable": {
+    "database_name": {
+      "type": "string",
+      "description": "The name of the database containing the current stage on which to grant privileges.",
+      "default": null
+    },
+    "per_privilege_grants": {
+      "type": "map(object({shares : list(string),roles : list(string)}))",
+      "description": "A map of privileges to authorized roles and shares. Privileges must be UPPER case.\n  This allows you to authorize extra roles/shares for specific privileges.",
+      "default": {}
+    },
+    "roles": {
+      "type": "set(string)",
+      "description": "Grants privilege to these roles.",
+      "default": null
+    },
+    "schema_name": {
+      "type": "string",
+      "description": "The name of the schema containing the current stage on which to grant privileges.",
+      "default": null
+    },
+    "shares": {
+      "type": "set(string)",
+      "description": "Grants privilege to these shares.",
+      "default": null
+    },
+    "stage_name": {
+      "type": "string",
+      "description": "The name of the stage on which to grant privileges.",
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used